core/src/main/groovy/noe/tomcat/configure/ConnectorAttributesTransformer.groovy

```diff
@@ -169,7 +169,7 @@ class ConnectorAttributesTransformer {
       if (connector.getSslPassword() != null && !connector.getSslPassword().isEmpty()) {
         attributes.put('SSLPassword', connector.getSslPassword())
       }
-      if (connector.getSslEnabledProtocols() != null && connector.getSslEnabledProtocols()) {
+      if (connector.getSslEnabledProtocols() != null) {
```

Comment:
Why?

Reply:
If user tried to set this value to `false` it will never be changed. I'm sure I placed this part there by wrong copy-paste

Reply:
I see, good point.